examples/resources/genesyscloud_telephony_providers_edges_extension_pool/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "genesyscloud_telephony_providers_edges_extension_pool" "test_extension_pool" {
+  start_phone_number = "1000"
```

Comment:
There's a mismatch between the `start_phone_number` and `end_phone_number` values. They're defined in the schema as `start_number`/`end_number` and used in the tests as `start_phone_number`.
Also they are `startNumber` and `endNumber` in the data source example.

Using `start_phone_number`/`end_phone_number` would be best as it would align with DID pools.

Reply:
Thank you for the review. I have resolved this is last commit.
I didn't use phone_number as I wanted to make a clear distinction between phone_number (I see this as a number reachable from outside) and number (extension only usable inside). If you catch my reasoning ;-)